man/docker-commit.1.md

```diff
@@ -17,6 +17,7 @@ CONTAINER [REPOSITORY[:TAG]]
 Create a new image from an existing container specified by name or
 container ID.  The new image will contain the contents of the
 container filesystem, *excluding* any data volumes.
+[Read more about valid image names and tags](https://docs.docker.com/reference/commandline/tag/).
```

Comment:
It may be better to refer to the "docker tag" man page, instead of a link to the documentation. People using the man pages usually don't want to open a browser for this, for example;

```Markdown
Refer to **docker-tag(1)** for more information
about valid image- and tag names.
```